test/integration/rules/aria-required-children/aria-required-children.html

```diff
@@ -76,8 +76,18 @@
   <div aria-busy="true"></div>
 </div>
 <div role="suggestion" id="fail10" aria-busy="false"></div>
+<div role="list" id="fail11">
+  <li>Item 1</li>
+  <span role="link">Item 2</span>
+</div>
+<div role="list" id="fail12">
+  <div aria-live="polite">
+    <div role="listitem">List item 1</div>
+    <div role="listitem">List item 2</div>
+  </div>
+</div>
 
-<div role="list" id="incomplete10">
+<div role="list" id="fail13">
```

Comment:
I don't think this should fail. Could keep incomplete, or pass. I'd guess @dylanb would rather see us just passing here.

Reply:
There's not a current way to make an exception to this since it's using a global aria attribute